backend/tests/unit/apps/owasp/management/commands/owasp_backfill_activity_events_test.py

```
@@ -0,0 +1,237 @@
+        mock_qs = MagicMock()
+        mock_qs.count.return_value = 2
+        mock_qs.__getitem__.return_value.iterator.return_value = iter([mock_obj1, mock_obj2])
+
+        command = Command()
+        command.stdout = MagicMock()
+        command.backfill_objects(mock_qs, 0, "issues", str)
+
+        assert mock_activity_event_class.update_data.call_count == 2
+
+    @patch("apps.owasp.management.commands.owasp_backfill_activity_events.ActivityEvent")
+    def test_backfill_objects_respects_offset(self, mock_activity_event_class):
```

Ruff (0.16.0)

[warning] Unused method argument: `mock_activity_event_class`

(ARG002)